packages/babel-helper-hoist-variables/src/index.ts

```diff
@@ -19,7 +19,7 @@ const visitor = {
     if (state.kind === "let") path.skip();
   },
 
-  Function(path: NodePath) {
+  FunctionParent(path: NodePath) {
     path.skip();
```

Comment:
This might actually be a bugfix?

```js
let x = async do {
  class A {
    static {
    	var a = 2;
    }
  }
}
```

since we use `hoistVariables` for asycn do expressions, I'd expect `var` here to be hoisted before this PR (but I didn't test it)

Reply:
It _will_ be a bugfix when we mark `StaticBlock` as `FunctionParent`, but I tend to land it in another PR with new tests.

Reply:
Oh ok :+1: